SWIPs/swip-honey_to_money.md

```diff
@@ -7,51 +7,66 @@ Type: Standards track
-
-In the current implementation, honey prices are hardcoded. When this price in the codebase changes, it does not affect nodes running a previous release; an old release expects a settlement of honey to be worth a different amount in currency. Eventually, this will result in nodes who run different versions of the software to disconnect from each other as the difference in prices will cause balances to be updated differently on each node. This SWIP proposes an on-chain price oracle to enable an atomic update process of honey prices without requiring nodes to update their source-code.
+This update process should be atomic: either all nodes upgrade or none. If Swarm would update the price by releasing a new version of the source code (as is currently the case), this would be violated as it is not possible to guarantee that all nodes are on the same release. By externalizing the price to an oracle, Swarm completely decouples its core functions from price discovery. By building support for price oracles, it will be possible to coordinate price changes across the network and set the stage for future experimentation with various mechanisms for setting prices, from DAOs, to bonding curves, to fixed prices. 
 
 ## Specification
-* The Swarm source code will reference the address of a `HoneyToMoney` price oracle
-* The price oracle implements the `HoneyToMoney` interface (to-be specified) 
-* The `HoneyToMoney` contract will be initially owned by an M/N multi-sig of Swarm developers and stakeholders.
-* Before sending a payment (denominated in currency), a node will query the `HoneyToMoney` price oracle to get the most up-to-date price conversion. 
-* Upon receiving a payment, nodes will determine the honey amount paid by looking up and dividing by the currently valid price from the oracle. This might cause payment imbalances between nodes, as the oracle might be queried one or several blocks before the transaction was sent. To avoid this situation, we propose that nodes include a payload with the transaction, specifying the time at which the oracle was queried.
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for the current Swarm platform and future client implementations.-->
+* An oracle is reachable at one specific endpoint and this endpoint is hardcoded in the Swarm source-code. 
+* No other factor besides time influences the answer which an oracle gives. That is: if nodes query the oracle at the same time, the oracle gives the same answer. 
```

Comment:
this is strictly correct, but it might be a little misleading, while true that at one point in time the oracle should give the same answer to queries by any number of nodes, we are glossing over the fact that these answers given by the oracle will be set (at least in part) manually.

Reply:
I don't understand what you mean here. Can you explain more?

Reply:
of course.

the way i understand it: the factor which directly influences the answer an oracle gives is those who govern it, through the values they set manually for these answers.

my point is that this is somewhat omitted by the phrase: `No other factor besides time influences the answer which an oracle gives`. i see time as a more indirect factor, so to speak.

the phrase might suggest that time causes a fluctuation of these values which is not manually induced... does it?